local.properties.example

```diff
@@ -8,4 +8,3 @@ io.getstream.chat.apiKey=
 io.getstream.chat.apiSecret=
 io.getstream.chat.url=https://chat.stream-io-api.com
 io.getstream.chat.timeout=10000
-# io.getstream.chat.test.logging=false
```

Comment:
maybe make it true by default

Reply:
Log is too verbose for now, so it's better clean it up before enabling by default

Reply:
good enough then